### PR TITLE
Kay D bowling challenge

### PR DESCRIPTION
**Pull request by kaykakaraka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,10 +2,33 @@
 Bowling Challenge
 =================
 
-* Feel free to use google, your notes, books, etc. but work on your own
-* If you refer to the solution of another coach or student, please put a link to that in your README
-* If you have a partial solution, **still check in a partial solution**
-* You must submit a pull request to this repo with your code by 9am Monday week
+This is a scorecard programme. The user enters their rolls as an array of arrays, with each array representing a frame, and receieve their score back.
+
+## EXAMPLE INPUT / OUTPUT
```
Comment (Jenniered) [inline README.md:7]: Like these examples included in your Readme.

File: scorecardIntegration.test.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+const Scorecard = require('./scorecard');
+
```
Comment (Jenniered) [inline scorecardIntegration.test.js:2]: Large range of tests covering the various possibilities of scores, including edge cases.

Comment (Jenniered) [reply scorecardIntegration.test.js:2]: Should there be unit tests for scorecard and scorecalculator?

File: ScoreCalculator.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+  giveTotal() {
+    this.findSparesAndStrikes();
+    if (this.bonusForSpares.length > 0) { this.addSpareBonus() };
+    if (this.bonusForStrikes.length > 0) { this.addStrikeBonus() };
+    this.scores.map((frame) => { 
+      this.total += frame.reduce((total, amount) => total += amount) 
+    });
+    return this.total;
+  }
+
```
Comment (Jenniered) [inline ScoreCalculator.js:18]: nice and clear function names

**Review by Jenniered — COMMENTED** [review]
Impressed by the very many commits highlighting your TDD process! And nice, concise code.
